StreamChatClient_v3/Workers/EventObservers/EventObserver_Tests.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient_v3
+import XCTest
+
+final class EventObserver_Tests: XCTestCase {
+    var notificationCenter: NotificationCenter!
+    var eventToDeliver: HealthCheckEvent!
+    var eventNotification: Notification {
+        .init(newEventReceived: eventToDeliver, sender: self)
+    }
+
+    // MARK: - Setup
+
+    override func setUp() {
+        notificationCenter = NotificationCenter()
+        eventToDeliver = HealthCheckEvent(connectionId: .unique)
```

Comment:
please also call `super` in `setUp` and `tearDown`

Reply:
fixed in c79a9c3ebd4f36f3dbb1bcc63f4b4a544765467a